### DISCUSSION TITLE
Explorer Status Bar issue since old version of Explorer Patcher

### DISCUSSION THREAD

**Discussion by GoldUltima** [valinet/ExplorerPatcher#519] [Q&A]
Since installing an old version of ExplorerPatcher, I've had an issue where there are two copies of the Status Bar in Explorer. One is an old grey version and the other is the newer Windows 11 version. I can turn the Windows 11 version on and off via the options menu or the registry but I cannot disable the older looking grey one. I've tried uninstalling ExplorerPatcher previously or updating to the newer version but it hasn't helped at all. Is there any known solution for this?
![Explorer](https://user-images.githubusercontent.com/95652962/144914369-a41471cb-6bca-4aa7-b5e0-e123e09bf32c.png)

**Comment by valinet** [comment]
1. Don’t use old versions.
2. That’s probably ClassicShell or something like that.

**Comment by Anixx** [comment]
This is not related to Explorer Patcher, it is Open Shell.
